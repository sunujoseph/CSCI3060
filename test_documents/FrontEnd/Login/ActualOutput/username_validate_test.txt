Enter Username: 
Error: Invalid Username
Enter Username: 