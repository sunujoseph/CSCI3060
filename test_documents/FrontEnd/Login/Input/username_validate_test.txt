login
user12345678912345678