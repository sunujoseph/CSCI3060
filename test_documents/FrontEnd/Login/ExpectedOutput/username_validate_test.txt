Enter Username: 
Error: Invalid username